pyroclast/common/visualizable.py

```diff
@@ -0,0 +1,94 @@
+import abc
+import numpy as np
+import tensorflow as tf
+
+
+class VisualizableMixin(abc.ABC):
+    """Interface Model interface.
+
+    Provides functionality for various calculations and
+    visualizations. Should be used as a mixin in conjunction with
+    tf.Module.
+
+    """
+
+    @abc.abstractmethod
+    def classify(self, x):
+        """Calculates output logits given a set of inputs.
```

Comment:
what is a "set of inputs"?

Reply:
True. This should probably just be a tensor.